plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -804,14 +854,19 @@ function OPDSBrowser:showDownloads(item)
             end
         end
     end
-    if (#type_buttons % 2 == 1) then -- we need even number of type buttons
-        table.insert(type_buttons, {text = ""})
+    -- handles adding download button(s)
+    if (download_buttons ~= {} and #download_buttons > 0) then
```

Comment:
Isn't it the same?

Also please locate download buttons by two in a row, as it was.

Reply:
The way it was before did not work, the separator would still appear if download_buttons was empty. The >0 check fixes that issue. 

As for the second thing, how do you mean "as it was"?

Reply:
https://github.com/koreader/koreader/blob/ee35257f997ec3cc719d23212428d39b4173129d/plugins/opds.koplugin/opdsbrowser.lua#L810-L812

Reply:
Oh, I forgot to put that back in. Thanks!